include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -1087,14 +1029,14 @@ struct buffered_piece_collection
             }
             if (detail::disjoint::disjoint_point_box(point,
                                                      original.m_box,
-                                                     d_pb_strategy_type()))
+                                                     m_strategy))
             {
                 continue;
             }
 
             int const geometry_code
-                = detail::within::point_in_geometry(point,
-                    original.m_ring, m_point_in_geometry_strategy);
+                = detail::within::point_in_geometry(point, original.m_ring,
+                                                    m_strategy);
```

Comment:
I don't know, I tend to align function arguments with the rest of the arguments if I can.

What exactly do you think should've been used here?
```
int const geometry_code
    = detail::within::point_in_geometry(point, original.m_ring,
        m_strategy);
```
```
int const geometry_code
    = detail::within::point_in_geometry(point,
        original.m_ring, m_strategy);
```
```
int const geometry_code
    = detail::within::point_in_geometry(
        point, original.m_ring, m_strategy);
```
The last one looks ok as well. So I can change it to the last one if you like.

And in this case this is probably also possible:
```
using detail::within::point_in_geometry;
int const geometry_code
    = point_in_geometry(point, original.m_ring, m_strategy);
```
since the same number of lines is used.

Or do you think about something else?

Reply:
Thanks for the details, I was also aligning function arguments with the rest of the arguments, but I am somehow convinced by https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers to wrap the lines (even this is not explicitly written there) so I am ok with any of your suggestions above.

Reply:
By the way, if it is possible that in future the library will achieve a compromise on using `clang-format` to ensure the consistent code format then, I think, it may be a good idea to pick the ones that `clang-format` can do best.

Reply:
I looked again and it seems that after putting all of the characters in one line here the line is shorter than some of the other lines in this file. Plus even in the guidelines we define max length as 100 characters. So I think this is the way to go here.

Personally I don't have any preference regarding the formatting besides looking good and allowing to easily catch the sense of an expression. I guess that in various contexts the best formatting would look differently. In any case we should probably aim for the least amount of lines. In case of a function call we should keep the parameters in one block whatever it means. Closer to the max line length this could mean putting all of the parameters at the next line. If there were more space this could mean aligning the parameters to the first one, etc. The number of parameters will also matter because putting each of them at subsequent line will cause the lines bloat. The context will also matter because a function call somewhere in a block should probably be treated differently than in the `if` statement condition. So I'm in general sceptical about automatic formatting and I'd simply leave that to the programmer. But maybe clang-format is intelligent enough? IDK.

Reply:
> I don't have any preference regarding the formatting besides looking good

Generally, the only preference I've got is consistency.

> In any case we should probably aim for the least amount of lines.

👍 

> I'm in general sceptical about automatic formatting and I'd simply leave that to the programmer. 

The `clang-format` is not ideal but it get close to almost any formatting style, depending on how sophisticated is team's preferences are. The compelling feature of `clang-format` is that it allows to achieve consistency with almost zero effort and it also helps me to review work by new contributors quicker as I don't have to explain over and over how to format, just point to `.clang-format` file and voila ;) 

Anyway, I meant a brief digression and not to open whole style debate, apologies.

Reply:
Ok, I understand, no need to apologise.